src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -350,7 +350,7 @@ void SettingsLoader::FinalizeLayering()
     if (userSettings.globals->EnableColorSelection())
     {
         const auto json = _parseJson(EnableColorSelectionSettingsJson);
-        const auto globals = GlobalAppSettings::FromJson(json.root);
+        const auto globals = GlobalAppSettings::FromJson(json.root, OriginTag::Generated);

```

Comment:
This one is weird, there's an argument here that the `OriginTag` should be `InBox`?